test/mocks/APWine.sol

```diff
@@ -11,6 +11,7 @@ contract APWine {
     uint256 public tokenOutCalled;
     uint256 public minimumAmountCalled;
     address public toCalled;
+    address public ownerCalled;
```

Comment:
no no. these are not in pattern. look at the mock Erc20. 

1. `fooCalled` is always the name of a method that was called - not the name of an argument.
2. depending on the number of args to the method the stored value can be:

    * one arg -> the value is the arg
    * two args -> a mapping whose key is the first arg and the val is the 2nd
    * 2+ args -> a mapping whose key is arg 1 and the mapping holds args 2 -> N by name